src/http/response.ts

```diff
@@ -19,11 +19,13 @@ export enum LkErrorKey {
   // Business and access errors
   FEATURE_DISABLED = 'feature_disabled',
   UNAUTHORIZED = 'unauthorized',
+  LDAP_BIND_ERROR = 'ldap_bind_error',
```

Comment:
This error key is actually no longer used. See https://github.com/Linkurious/linkurious-server/pull/4649#discussion_r1581249386

So we don't need to add this entry.